tests_integration/testing_utils/project_fixtures.py

```diff
@@ -0,0 +1,137 @@
+from textwrap import dedent
+
+import pytest
+
+from tests.nativeapp.factories import (
+    ProjectV2Factory,
+    ApplicationPackageEntityModelFactory,
+    ApplicationEntityModelFactory,
+)
+
+
+MANIFEST_BASIC = dedent(
```

Comment:
Can't we use ManifestFactory?
```"manifest.yml": ManifestFactory(),```

Reply:
Might have to use name `setup.sql` for set up script (instead of `setup_script.sql`) but the rest should work fine.